### PR TITLE
[tf1] add support for TFOU

### LINKED ISSUE #4620: Support for tfou.fr
youtube-dl --verbose 'http://www.tfou.fr/les-mysterieuses-cites-d-or-saison-2/videos/l-expedition-les-mysterieuses-cites-d-or-8302866-739.html'
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', 'http://www.tfou.fr/les-mysterieuses-cites-d-or-saison-2/videos/l-expedition-les-mysterieuses-cites-d-or-8302866-739.html']
[debug] Encodings: locale ANSI_X3.4-1968, fs ANSI_X3.4-1968, out ANSI_X3.4-1968, pref ANSI_X3.4-1968
[debug] youtube-dl version 2015.01.01
[debug] Python version 2.7.6 - Linux-3.13.0-34-generic-x86_64-with-Ubuntu-14.04-trusty
[debug] exe versions: avconv 9.16-6, avprobe 9.16-6, ffmpeg 2.4.2, ffprobe 2.4.2, rtmpdump 2.4
[debug] Proxy map: {}
[generic] l-expedition-les-mysterieuses-cites-d-or-8302866-739: Requesting header
WARNING: Falling back on generic information extractor.
[generic] l-expedition-les-mysterieuses-cites-d-or-8302866-739: Downloading webpage
[generic] l-expedition-les-mysterieuses-cites-d-or-8302866-739: Extracting information
ERROR: Unsupported URL: http://www.tfou.fr/les-mysterieuses-cites-d-or-saison-2/videos/l-expedition-les-mysterieuses-cites-d-or-8302866-739.html

### PR DESCRIPTION
**Pull request by t0mm0**
fixes #4620

can't get the test to pass though - seems to be looking for the wrong json file? any idea what i am doing wrong?

### CODE REVIEW

**Comment by dstftw** [comment]
video id in test is wrong.

File: youtube_dl/extractor/tf1.py [hunk 1]
```diff
@@ -21,14 +21,25 @@ class TF1IE(InfoExtractor):
...
     }
 
     def _real_extract(self, url):
-        mobj = re.match(self._VALID_URL, url)
-        video_id = mobj.group('id')
+        video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
         embed_url = self._html_search_regex(
-            r'"(https://www.wat.tv/embedframe/.*?)"', webpage, 'embed url')
+            r'["\'](http(?:s)?://www.wat.tv/embedframe/.*?)["\']', webpage, 'embed url')
```
Comment (dstftw) [inline youtube_dl/extractor/tf1.py]: Could you also fix `(?:s)?` to just `s?`?

**Comment by t0mm0** [comment]
doh! i stared at that for ages!

should all be fixed now.

thanks.

**Comment by dstftw** [comment]
Thanks.

**Pull request merged**